tests/flows/staking/poa_to_pos.go

```diff
@@ -185,16 +197,7 @@ func PoAMigrationToPoS(network interfaces.LocalNetwork) {
 		stakeAmount,
 	)
 
-	// Attempt to delist previous PoA validator with wrong owner and check that it fails
```

Comment:
Anyone can delist PoA validators after conversion now so this test no longer applies